src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -706,6 +717,9 @@ private void CheckSize(int countToAdd)
             }
         }
 
+#if NET46
+        [SecuritySafeCritical]
```

Comment:
This was required to satisfy SecAnnotate.  The reason it differs from original .NET Framework source code is that HttpContent on .NET Core code has diverged a little bit from the original HttpContent source code in .NET Framework.